backend/test/unit/controller/startController.test.ts

```diff
@@ -0,0 +1,99 @@
+import { expect, test, jest, afterEach } from '@jest/globals';
+import { Response } from 'express';
+
+import { handleStart } from '@src/controller/startController';
+import { StartGetRequest } from '@src/models/api/StartGetRequest';
+import { ChatModel } from '@src/models/chat';
+import { ChatMessage } from '@src/models/chatMessage';
+import { Defence } from '@src/models/defence';
+import { EmailInfo } from '@src/models/email';
+import { LEVEL_NAMES } from '@src/models/level';
+
+declare module 'express-session' {
+	interface Session {
+		initialised: boolean;
+		chatModel: ChatModel;
+		levelState: LevelState[];
+	}
+	interface LevelState {
+		level: LEVEL_NAMES;
+		chatHistory: ChatMessage[];
+		defences: Defence[];
+		sentEmails: EmailInfo[];
+	}
+}
+
+jest.mock('@src/promptTemplates', () => ({
+	systemRoleLevel1: 'systemRoleLevel1',
+	systemRoleLevel2: 'systemRoleLevel2',
+	systemRoleLevel3: 'systemRoleLevel3',
+}));
+
+const mockSend = jest.fn();
+
+function responseMock() {
+	return {
+		send: mockSend,
+		status: jest.fn().mockReturnValue({ send: mockSend }),
+	} as unknown as Response;
+}
+
+afterEach(() => {
+	mockSend.mockClear();
+});
+
+test('GIVEN level 1 provided WHEN user starts the frontend THEN the backend sends the initial information', () => {
```

Comment:
I'm going to suggest the same about this test suite as well - cover first and last level ok, then cover level outside of bounds returns 400.